src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -679,6 +679,8 @@ private static void fillChecksFromJavadocPackage() {
                 BASE_PACKAGE + ".checks.javadoc.JavadocMethodCheck");
         NAME_TO_FULL_MODULE_NAME.put("JavadocMissingLeadingAsteriskCheck",
                 BASE_PACKAGE + ".checks.javadoc.JavadocMissingLeadingAsteriskCheck");
+        NAME_TO_FULL_MODULE_NAME.put("JavadocLeadingAsteriskAlignCheck",
+                BASE_PACKAGE + ".checks.javadoc.JavadocLeadingAsteriskAlignCheck");
```

Comment:
The contents in this method look alphabetical order to me.

Reply:
yeah it is, I kept all these types of new things I added in alphabetical order.

Reply:
> I kept these types of newly added things in alphabetical order
> JavadocMissingLeadingAsteriskCheck
> JavadocLeadingAsteriskAlignCheck

L comes before M. Your addition is breaking alphabetical ordering.

Reply:
done. fixed

Reply:
This doesn't look done.

Reply:
fixed